code/src/GridAPI/Pagination.ts

```diff
@@ -160,22 +171,35 @@ namespace GridAPI.Pagination {
      * @export
      * @param {string} gridID
      * @param {number} n
-     * @returns {*}  {void}
+     * @returns {*}  {string} Stringified JSON structure containing error message and code, and success boolean
      */
-    export function MoveToPage(gridID: string, n: number): void {
+    export function MoveToPage(gridID: string, n: number): string {
         PerformanceAPI.SetMark('Pagination.MoveToPage');
+        let output = '';
 
-        if (!OSFramework.Helper.IsGridReady(gridID)) return;
-        const grid = GridManager.GetGridById(gridID);
+        if (!OSFramework.Helper.IsGridReady(gridID)) {
+            let returnMessage = new OSFramework.OSStructure.ReturnMessage();
+
+            returnMessage = {
+                value: -1,
+                isSuccess: false,
+                message: 'Grid not found',
+                code: OSFramework.Enum.ErrorCodes.CFG_GridNotFound
+            };
```

Comment:
Same as above...

Reply:
That's actually not overriding or is it? If you don't instantiate the ReturnMessage before you assign the values you won't have the intellisense